pallets/gear-bank/src/lib.rs

```diff
@@ -430,8 +449,15 @@ pub mod pallet {
 
             let value = Self::withdraw_gas_no_transfer(account_id, amount, multiplier)?;
 
-            Self::withdraw_on_finalize(to, value)
-                .unwrap_or_else(|e| unreachable!("qed above: {e:?}"));
+            Self::withdraw_on_finalize(to, value).unwrap_or_else(|e| {
+                let err_msg = format!(
+                    "Pallet::spend_gas_to: withdraw on finalize failed. \
+                        Account id - {account_id:?}, amount - {amount}. Got error - {e:?}"
```

Comment:
same additional data required here as for the `withdraw` method result handling

Reply:
Fixed